<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>estoque</class>
 <widget class="QDialog" name="estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>971</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_novoproduto">
    <attribute name="title">
     <string>Novo Produto</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>941</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>9</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Código do Produto</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="cod_prod"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Descrição produto</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="desc_prod"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Fornecedor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fornecedor"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Valor de compra</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="valor_compra"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Quantidade de compra</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="qtd_estoque"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Valor de venda</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="valor_venda"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="novo_produto">
          <property name="text">
           <string>Novo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gravar">
          <property name="text">
           <string>Gravar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_gestaoprod">
    <attribute name="title">
     <string>Gestão Produtos</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
